docs/source/tutorial/visualise_pipeline.md

```diff
@@ -106,8 +106,8 @@ Kedro-Viz aims to help users communicate different aspects of their workflow thr
 We have also used the Plotly integration to allow users to [visualise metrics from experiments](../logging/experiment_tracking.md).
 
 
-```eval_rst
-  .. note:: Kedro's Plotly integration only supports `Plotly Express <https://plotly.com/python/plotly-express/>` charts.
+```{note}
+Kedro's Plotly integration only supports `Plotly Express <https://plotly.com/python/plotly-express/>` charts.
```

Comment:
```suggestion
Kedro's Plotly integration only supports [Plotly Express](https://plotly.com/python/plotly-express/) charts.
```
https://kedro.readthedocs.io/en/0.17.7/03_tutorial/05_visualise_pipeline.html is rendered wrpong currently